src/steps/migrate_translations.ts

```diff
@@ -3,5 +3,10 @@ export default async options => {
   const dest = options.get("dest");
   const editor = options.get("editor");
   // Copy all translation files across
-  editor.copy(`${src}/translations/*`, `${dest}/src/translations/`);
+  try {
```

Comment:
I wonder whether it's worth extracting a spec helper for these copy operations, something like `tryCopy` that will wrap the `try... catch`?  It only appears 3 times, so not a huge deal, but could be slightly DRY'er

Reply:
agree. Its usage will likely to increase in the future so I'll extract it to `utils.ts`.